templates/package/security_advisories.html.twig

```diff
@@ -32,6 +32,7 @@
                                 <div class="row">
                                     <div class="col-sm-8 col-lg-9">
                                         <h4 class="font-bold">
+                                            {% if advisory.severity %}[{{ advisory.severity|upper }}]{% endif %}
```

Comment:
Can you please rebase due to https://github.com/composer/packagist/issues/1392#issuecomment-1782453560 and make sure to add the severity display as well somehow to the new security_advisory.html.twig template

Reply:
rebased and extracted the security advisory list into a shared template